src/app/utilities/common.ts

```diff
@@ -42,14 +43,28 @@ export const throttle = (
   };
 };
 
-export const createMetadata = ({
-  title,
-  keywords,
-  description,
-  canonical,
-  type,
-  images,
-}: MetadataParams) => {
+export const createMetadata = ({ key, params }: MetadataKeyValuePair) => {
```

Comment:
Instead of one parameter, accept two parameters please. Passing the second parameter would be optional for this function. Name the second parameter something that signifies that it can 'extend' the base configuration.